src/Util/ObjectAclManipulator.php

```diff
@@ -13,10 +13,13 @@
 
 namespace Sonata\DoctrineORMAdminBundle\Util;
 
+use Doctrine\ORM\EntityManager;
 use Sonata\AdminBundle\Admin\AdminInterface;
 use Sonata\AdminBundle\Exception\ModelManagerException;
 use Sonata\AdminBundle\Security\Handler\AclSecurityHandlerInterface;
 use Sonata\AdminBundle\Util\ObjectAclManipulator as BaseObjectAclManipulator;
+use Sonata\DoctrineORMAdminBundle\Model\ModelManager;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
```

Comment:
I think `Doctrine\Persistence\ManagerRegistry` is enough?

Reply:
I don't know at all what is this Manager registry, I copied the code of DoctrineMongo made by @franmomu  I think.
So sure, I can try